cli/provider/cmd.go

```diff
@@ -644,3 +649,21 @@ func (c *CmdConfigurator) UpdateConfigData(defaultCfg config.Config) config.Conf
 	defaultCfg.Test.Mocking = c.cfg.Test.Mocking
 	return defaultCfg
 }
+
+func (c *CmdConfigurator) RemoveInternalParameters(defaultCfg config.Config) (map[string]interface{}, error) {
+	var cfgMap map[string]interface{}
+	err := mapstructure.Decode(defaultCfg, &cfgMap)
+
+	if err != nil {
+		utils.LogError(c.logger, err, "error decoding struct")
+		return nil, errors.New("error decoding struct")
+	}
+
+	fieldsToRemove := []string{"generateGithubActions", "keployNetwork", "keployContainer", "inCi", "cmdType"}
```

Comment:
For this you can create a function in config package itself. Once user adds something in internal config it should automatically be deleted from the config. We should not mention it again in another place